components/download_images/fondant_component.yaml

```
@@ -1,6 +1,6 @@
 name: Download images
 description: Component that downloads images based on URLs
-image: ghcr.io/ml6team/download_images:dev
+image: ghcr.io/ml6team/download_images:latest
```

```suggestion
image: ghcr.io/ml6team/download_images:dev
```

The images on main should be fixed to `dev`, which corresponds to the latest main version. `latest` corresponds to the latest release.

---

right forget to revert this back!